packages/scripts/config/webpack.config.js

```diff
@@ -245,21 +254,69 @@ const config = {
 			},
 		],
 	},
+	stats: {
+		children: false,
+	},
+};
+
+// WP_DEVTOOL global variable controls how source maps are generated.
+// See: https://webpack.js.org/configuration/devtool/#devtool.
+if ( process.env.WP_DEVTOOL ) {
+	baseConfig.devtool = process.env.WP_DEVTOOL;
+}
+
+if ( ! isProduction ) {
+	// Set default sourcemap mode if it wasn't set by WP_DEVTOOL.
+	baseConfig.devtool = baseConfig.devtool || 'source-map';
+	baseConfig.devServer = {
```

Comment:
Will two configs integrate seamlessly with a single dev server that uses shared setup?

Reply:
Good spot, this needs to be configured with different ports.

Reply:
It looks like devServer doesn't really work with multiple compilations: https://github.com/webpack/webpack-cli/issues/2408

I'll only add this to 1 config.